comment.go

```diff
@@ -0,0 +1,38 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import "fmt"
+
+// Commenter is used to attach comments to checks and assertions, and must be
+// implemented in order to provide extra information in case of failures.
+type Commenter interface {
+	// Comment returns a comment for the given error, which could be a proper
+	// test failure or a bad check. The returned text message is included in the
+	// failure log.
+	Comment(error) string
```

Comment:
Is it really worth passing the error in here? What kind of functionality are you thinking of here?
I'm not sure that the interface adds any particular value.

I think I'd keep things less abstract:

    // Comment represents a comment on a test failure.
    type Comment struct {
        format string
        args []interface{}
    }

    // Commentf ... etc
    func Commentf(f string, a ...interface{}) Comment {
        return Comment{f, a}
    }

Reply:
Done.